gallery/src/main.scss

```diff
@@ -9,41 +9,63 @@
 @import '~@sonatype/react-shared-components/scss-shared/nx-container-helpers';
 @import '~@sonatype/react-shared-components/scss-shared/nx-flexbox-helpers';
 
+.nx-page-sidebar.gallery-page-sidebar {
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+}
+
 .gallery-nav, .gallery-nav-links {
   @include container-vertical;
 }
 
-.gallery-nav-category {
-  @include semi-bold();
+.gallery-nav {
+  .nx-tree-view__trigger {
+    .nx-tree-view__text {
+      @include semi-bold();
 
-  color: $nx-text-color-light;
-  font-size: 16px;
-  margin-top: 16px;
-  margin-bottom: 12px;
+      color: #393939;
+      font-size: 16px;
+    }
+  }
 }
 
-.gallery-nav-link {
+.nx-tree-view__child.gallery-nav-link {
+  font-size: 16px;
+  height: 32px;
+  line-height: 32px;
+  margin-right: 0;
   margin-bottom: 4px;
   margin-left: 0;
 }
 
 .gallery-link {
+  border-radius: 6px;
+  box-sizing: border-box;
+  color: $nx-text-color-light;
```

Comment:
This isn't actually doing anything; it's getting overridden on the same style on `.nx-tree-view__child a`

Reply:
> This isn't actually doing anything; it's getting overridden on the same style on `.nx-tree-view__child a`

https://github.com/sonatype/sonatype-react-shared-components/pull/31/commits/6855d2021c7345f89bbd5293bf5a6c9c47094b0e